functests/4_latency/latency.go

```diff
@@ -428,6 +434,16 @@ func createLatencyTestPod(testPod *corev1.Pod, node *corev1.Node, logName string
 	err = pods.WaitForPhase(testPod, corev1.PodRunning, 2*time.Minute)
 	Expect(err).ToNot(HaveOccurred())
 
+	if runtime, _ := strconv.Atoi(latencyTestRuntime); runtime > 1 {
+		By("Checking actual CPUs number for the running pod")
+		limitsCpus := testPod.Spec.Containers[0].Resources.Limits.Cpu()
+		RequestsCpus := testPod.Spec.Containers[0].Resources.Requests.Cpu()
```

Comment:
this doesn't seem correct, being both `testPod.Spec.Containers[0].Resources.Limits` and `testPod.Spec.Containers[0].Resources.Requests` a `type ResourceList map[ResourceName]resource.Quantity`

Reply:
Updated.Thanks!